### ISSUE TITLE
Intermittently unable to start mongodb service

### ISSUE THREAD

**Issue by mattramsay-ice** [actions/runner-images#8195]
### Description

Intermittently, I am unable to start the mongodb service on the latest windows-2022 image (20230820.1.0), with the error being:

```
The service is not responding to the control function.

More help is available by typing NET HELPMSG 2186.
```

This occurs after ~2mins of waiting for the service to start.

Also, worth noting that I am enabling the service prior to starting it.

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] Windows Server 2019
- [X] Windows Server 2022

### Image version and build link

20230820.1.0

### Is it regression?

yes, this worked consistently on version 20230804.1.0

### Expected behavior

Mongodb service consistently starts successfully 

### Actual behavior

Mongodb service fails to start, errors after ~2mins with the following:

```
The service is not responding to the control function.

More help is available by typing NET HELPMSG 2186.
```

### Repro steps

Include the following step in an azure pipeline (windows-2022)  
```
 - script: | 
        sc config "MongoDb" start=demand
        net start mongodb 
      displayName: Start mongodb
```

**Comment by vpolikarpov-akvelon** [comment]
Hey @mattramsay-ice. I tried reproduce this in my environment, but didn't succeed. The script you provided works for me. Do you experience the problem when this script is the only step in your pipeline? If not, what are other steps?

**Comment by mattramsay-ice** [comment]
Of course, every single run I've done since this issue has been raised has not reproduced the error stated. Nothing has changed in my pipeline yml, so I'm at a loss.

Happy for this to be closed and thanks for looking into it.
